.storybook/preview.ts

```diff
@@ -27,11 +27,8 @@ addParameters({
   },
   layout: "centered",
   options: {
-    storySort: (a, b) => {
-      const sectionA = a[1].id.split("-")[0];
-      const sectionB = b[1].id.split("-")[0];
-
-      return sectionB.localeCompare(sectionA);
```

Comment:
@jcfranco I think this would remove the current logic that orders by local file name. For example, mdx files are ordered by number in their local file name in order to specify a specific order. If we remove this, can we come up with a way to order them?

Reply:
Maybe we should add a comment about this so its not messed up?

Reply:
I think you've brought this up before and I totally forgot. A comment would've helped me. 😅

Just for my understanding, could you clarify what would change if we remove this? All stories, including MDX files, are ordered the same as before:

### Overview (after)

<img width="216" alt="story-sort-after" src="https://user-images.githubusercontent.com/197440/95608813-8e6b0700-0a12-11eb-87cc-4a9a8ccbcf24.png">

### Sidebar (after)

<img width="125" alt="all-stories-after" src="https://user-images.githubusercontent.com/197440/95609315-5ca67000-0a13-11eb-85d5-39fe20e3c802.png">

Reply:
Oh ok maybe those are doing the same thing. If we can verify that the order is kept so that `Home` is always first (etc) then I think we're good to go

Reply:
Maybe we could dig into the source code to confirm?

Reply:
Awesome, thanks for clarifying! I'll double check one more time and will proceed to install if the order is the same. 💪😎